src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -1031,6 +1031,28 @@ export function getEnrichmentBarPlotData(
     );
 }
 
+export function compareByAlterationPercentage(
+    kv1: AlterationEnrichmentRow,
+    kv2: AlterationEnrichmentRow
+) {
+    const t1 = _.reduce(
+        kv1.groupsSet,
+        (acc, next) => {
+            acc = next.alteredPercentage > acc ? next.alteredPercentage : acc;
+            return acc;
+        },
+        0
+    );
+    const t2 = _.reduce(
+        kv2.groupsSet,
+        (acc, next) => {
+            acc = next.alteredPercentage > acc ? next.alteredPercentage : acc;
+            return acc;
+        },
+        0
+    );
+    return t2 - t1;
+}
```

Comment:
let's add a new line here

```suggestion
}

```